### ISSUE TITLE
Explanations

### ISSUE THREAD

**Issue by garry-ut99** [garry-ut99/garry-ut99#1]
As for https://github.com/uBlockOrigin/uAssets/issues/21305#issuecomment-1848581980 : I thought you were writting about adding a filter, to ryanbr, because it was looking like so, because you called his nickname, and your whole comment looks like you were talking to ryanbr, that's why I mentioned that it should be `:-abp-contains`, instead of `:has-text`, but now I've realized that maybe you were actually writing to the OP, and only called ryanbr "by the way". Anyway, I'm not sure who is to be blamed for the misunderstanding of context of your comment: either fault on my side: I misread your comment, or fault on your side: your comment was misleading, or the blame is shared.

**Comment by mapx-** [comment]
It's not a big deal, indeed the filter was for OP and then - as usually - I notified the maintainer.

**Comment by garry-ut99** [comment]
Not over and over, what is this moderation abuse show, it seems some guy finds whatever reason to hide my comments:
- https://github.com/uBlockOrigin/uAssets/issues/21321#issuecomment-1848984062
It was not my comment which was the duplicate, does he has a problem with his eyes?
Or did he prefer to hide my comment instead of moderator's? In this case, despite I'm not a moderator, 
I'm not a noob neither, he could just not hide any of them
- https://github.com/uBlockOrigin/uAssets/issues/21450#issuecomment-1859141399
if an user reads last duplicates, will easily notice : 
`https://github.com/uBlockOrigin/uAssets/issues/18928#issuecomment-1630469187`
 I'm not a noob neither, he could just not hide my comment

Also I don't post such comments too often, only sometimes, so in no way did I abuse, 
Nor are these threads with many comments to keep them cleaned up so much,
who and why is doing it , I feel disrespected, like someone does it intentionally to tease me 
or to feel better than others. Stop it or I stop contributing, I'm not a monkey in anybody's circus.

**Comment by MasterKia** [comment]
As for https://github.com/uBlockOrigin/uAssets/issues/21321, I didn't even see your comment at the time and just closed the issue. I made your comment visible now.

**Comment by garry-ut99** [comment]
Abuse of moderation again, another my comment has been hidden :
https://github.com/uBlockOrigin/uBlock-issues/issues/3090#issuecomment-1904737589

 I put a backup here :
<details>
<summary>Details</summary>

> https://github.com/uBlockOrigin/uBlock-issues/issues/3090#issuecomment-1904737589 : garry-ut99 :
> Cheap excuse or poor searching, I wonder how you were smart enought to bypass the issue template, and at the same time not smart enought to find many duplicate issues, but you're asking for a fish so nicely, that it seems everyone will forgive you anyway.

And then I've posted my reply (which might be hidden too, or removed), so here backup too:


> https://github.com/uBlockOrigin/uBlock-issues/issues/3090#issuecomment-1905630692 : garry-ut99 :
> To whoever has hidden my comment: stop disrespectfully hiding my comments, are you feeling better now? My comment was a peaceful/sarcastic reply, I just stated few facts and I let it go, because the guy was nice, but you are hallucinating/overreacting and you have had to hide it for whatever weird reasons you imagined in your head, if you prefer some noob, than a several years contributor, then maybe go and marry this noob.
> 
> Now what? Let me guess, you lack arguments, and you will provide (or not) some crappy reply, to which you will not let me respond to, like usual, and you gonna hide this comment too? Or even remove it? Or maybe lock the thread? Or ban me? I wonder who has hidden my comment this time, the previous time was here : https://github.com/garry-ut99/test/issues/1#issuecomment-1859283657

</details>

<hr>

01.06.2024:

> - https://github.com/uBlockOrigin/uAssets/issues/23932

Abuse of moderation again: **ghajini** I assume it was you since the "invalid" label and locked thread appeared at the same time: why have you acted like a p.... and disrespectfully locked the thread instead of converting it to a discussion? OP and me wanted to continue in Discussions, but you interferred us, could you stop interferring other users for no reason... Try to control your emotions. What is wrong with some uAssets guys? 

Curious whether @ stephenhawk8054 @ MasterKia do have any clue why their crew member: ghajini has acted like this. @ sankhababu don't worry about those uAssets guys

**Comment by garry-ut99** [comment]
The shit-show happens again: @ghajini I assume it was you since the "invalid" label and locked thread appeared at the same time: why have you acted like a prick and disrespectfully locked the thread instead of converting it to a discussion? OP and me wanted to continue in Discussions, but you interferred us, could you stop interferring other users for no reason... If you suffer from some mental/emotional instability problems you can always start taking some meds, take your meds bro, it will be the best for the community.  What is wrong with some uAssets guys? Some of them behave nasty, instead of locking threads they should lock themself in cages.

**Comment by ghajini** [comment]
hello
that issue report https://github.com/uBlockOrigin/uAssets/issues/23932
was invalid when you closely look 

also questions/support is available at https://www.reddit.com/r/uBlockOrigin/
also don't mention me unnecesarily, you want chit chat ; take that nasty/abusive/spammy talk in your repo, uassets is not place for spammy/rubbish talk

**Comment by garry-ut99** [comment]
Hey, you still didn't answer the main question: why you rudely locked the thread instead of transferring it to Discussions or to uBlock, is the question to hard for you to understand? The issue was looking like it might be some issue with uBO itself or with whatever else on OP's side, most of time in such situations such issues are not locked, but transferred to Discussions or to uBlock repository, to be continued and solved. You just acted rudely, and when I asked you why did you do it, you're evading answers, and still are continuing to act like this.

> ghajini: that issue report https://github.com/uBlockOrigin/uAssets/issues/23932 was invalid when you closely look

 I didn't intend to accuse you of attaching "invalid" label, only of locking the thread instead of transferring it.  I mentioned the label only because it mentioned your nickname, to make a logical assumption that it was also you who locked the thread, as "locking the thread" action has no sticked nickname to it, and sometimes it might happen that such two actions can be done by two different users, I wanted to have a strong argument that it was you doing the both actions, and nobody else.

> ghajini: also don't mention me unnecesarily, 

Unnecesarily? You acted rudely and I think we deserve explanations why you acted like this, hence I'm convinced I mentioned you for a valid reason, I also bringed in  your crew members to let them see what guy they have in their team.

> ghajini: you want chit chat ; take that nasty/abusive/spammy talk in your repo, uassets is not place for spammy/rubbish talk

What ? I'm getting lost, do you mean trash-talk here or in the thread on uAssets?
- trash-talk here is a result of your abusive/shitty/disrespectful action on uAssets, don't cry now just because you're being called appropriate words adapted to your shitty level, you deserved it
- but if you mean trash-talk in locked thread, I don't see any, only a technical discussion and a willingness to help others, the only trash and disruptive behavior in the the locked thread is your provocative action of locking the thread instead of simply transferring it, so I don't know what are you talking about

You seem to have emotional stability problems, you are full of anger, with oversized ego, your actions are disrespectful towards other members of community, and you're trying to shift the blame on them, just look in the mirror.

@ MasterKia I'm curious why did you react with confused emoji, if there is anything which is not clear for you in my comment, I'm will to explain it, if you need, but I think that after the current comment it's more clear now.

**Comment by garry-ut99** [comment]
> - https://github.com/uBlockOrigin/uBlock-issues/discussions/2981

@ **iam-py-test** if you don't take actual place in discussions nor you don't create comments / nor you don't help / nor you don't provide answers / solutions nor you don't make any contribution to a discussion, then gtfo, and go troll elsewhere with your thumb up emojis and stop sticking them to your idol (gorhill), stop licking & cleaning his arseh... each time he replies, because it disrespectful towards other users, and this is neither a porn tube nor facebook, you are useless in our discussions, you disturb the assessment of the situation of "who is right/useful and who is not", in a discussion, in a favor of one person cherry-picked by you, it's not the first time you are spamming threads with your emojis, you've been doing it for a long time, dozen of times, in dozen of issues/discussions, go troll elsewhere.

**Comment by garry-ut99** [comment]
Two more trolls (from other repositories), none of them provided any explanations, arguments, both of them are desperately out of arguments, they blame, troll and manipulate, the first guy desperately locked threads, the second guy even worse - desperately removed my comments:
- https://github.com/BurntSushi/ripgrep/discussions/2867 (the first troll)
- https://github.com/Genivia/ugrep/issues/416 (the second troll)
   - my comments removed by the second troll:
<details>
<summary>my comments removed by the second troll (SPOILER):</summary>

### 1

> garry-ut99: (unjustly removed comment) : https://github.com/Genivia/ugrep/issues/416#issuecomment-2278364348 ==> https://github.com/Genivia/ugrep/issues/416#event-13824422863 :
> 
> Maybe because you weren't clear enough, since you created two separate comments, it misled me into thinking that you wanted me to try the both methods separately (non-parallel and parallel), perhaps instead of blaming me immediately, you should have been edit your previous comment just like you did before, instead of creating another one, or to make it very clear that the both comments are directly connected. Optionally, I can agree that the blame for the context/wording misunderstanding lies on both sides.
> 
> > genivia-inc : Please read.
> 
> Please write clearly and don't blame. Optionally, I can agree that the blame lies on both sides, I also do understand you didn't like my reply, as it was created with a misassumption, and puts your method in a bad light, anyway your "please read" followed by closing thread, reminds me "please open a new issue" followed by locking thread, from the other rude guy, the difference between him and you is that he was rude since the beginning:
> 
> - https://github.com/BurntSushi/ripgrep/discussions/2867
> 
> > genivia-inc  : I think this answers your question.
> 
> Maybe, but theory doesn't always match reality, on my side I can be sure only after doing benchmarks, when I get time, but I don't know whether I'm interested anymore.
> 

> genivia-inc : https://github.com/Genivia/ugrep/issues/416#issuecomment-2278534747 : I've deleted the last message posted by @garry-ut99 because it was not in compliance with the [code of conduct](https://github.com/Genivia/ugrep/blob/master/CODE_OF_CONDUCT.md?rgh-link-date=2024-08-09T18%3A42%3A03Z).

So I've repasted it here above, because I don't see anything offensive in my previous comment, which could break the "code of conduct", only few words of truth, in the worst case, I only adapted my level to your level, so what is your problem then, it's not me who started.

You only provided a generic link to "code of conduct" without actually providing and proving which specific sentences of my previous comment break which specific points of "code of conduct", you proved nothing, yet you still removed my comment, accusing me of breaking "code of conduct". The irony is that it's your and the other guy's actions, which can be considered breaking of "code of conduct", how can you accuse one of breaking "code of conduct" after previously breaking it by yourself.

> genivia-inc :  Edit: screenshot evidence:

Since you're speaking of evidence, where is your evidence that I broke "code of conduct" in my previous comment..., what logic or mathematical formula you use to choose for which cases to present evidence and for which to not...there is no consistency in this logic, before removing my comment again, prove and show it, of course allow me to reply to it (to agree or disagree with your "evidence"), otherwise it will be nothing more than desperate and deliberate trolling / censorship, and it will mean you are desperately out of arguments, and I'll leave this place forever, go ahead, if this is what you want, I don't care.

<hr>

### 2

> genivia-inc : https://github.com/Genivia/ugrep/issues/416#issuecomment-2278749375 : To prove it to you, my suggestion was to use `tee` and a named pipe and put in a script file e.g. `teetest.sh`

To prove waht exactly....you can prove your algorithm, but certainly not the drama part, as there was no such suggestion in your first comment, as it doesn't contain words: "tee" or "pipe"  (CTRL + F in browser returns nothing), you only suggested it in your second comment, which I didn't realize it was directly connected/related to the previous comment, not to say I didn't even had time back then to try method proposed in your second comment, as I was tired and busy already.

<hr>

### 3

> genivia-inc : https://github.com/Genivia/ugrep/issues/416#issuecomment-2278749375 : my suggestion was to use `tee` and a named pipe and put in a script file e.g. `teetest.sh`

I didn't read the thread yesterday (I did a little break of reading), so I didn't see your last comment yesterday, nor that you removed my comment, I see it only now, however I spent yesterday few hours and was testing your method proposed in your second comment, also today I also read your last comment, but unfortunately you proposed me solution for Linux, and I failed when trying to port your Linux command to Windows batch version (I also heard about PowerShell way), so if you want me to confirm that your method works on Windows, then maybe you can fix it (or not, I don't care, it's your choice), I spent enough time on this:
- Tee for Windows: https://github.com/dEajL3kA/tee-win32
- windows-named-pipe-utils : https://github.com/psmay/windows-named-pipe-utils


<details>
<summary>input_to_output.bat (SPOILER):</summary>

```
@echo off  
set startTime=%time%
REM main path to RAMDISK
set path=F:
set path_ug=ugrep-windows-x64
set path_tee=_input_to_output\tee-win32.2023-11-27
set path_pipes=_input_to_output\windows-named-pipe-utils
REM input_file size 661 MB / 32237207 lines / each line is domain name / sorted a-z order / duplicates removed
set input_file=input_file.txt
set temp_file=temp_file.txt
set output_file_A=output_file_A.txt
set output_file_B=output_file_B.txt
set phrase="someone"
set matched="^[0-9]+:(.*)"
set unmatched="^[0-9]+-(.*)"



REM METHOD 1
REM Standard synchronic method for Windows: 3.28s total 
REM using --encoding=UTF-8 and -a, otherwise it throws error: "binary file"
REM %path_ug%\ug --encoding=UTF-8 -ahrP --stats  %phrase% "%path%\%input_file%" > "%path%\%output_file_A%"
REM Searched 1 file in 0.59 seconds with 7 threads: 1 matching (100%)
REM Searched 32237206 lines: 509 matching (0.001579%)
REM %path_ug%\ug --encoding=UTF-8 -ahrP -v --stats  %phrase% "%path%\%input_file%" > "%path%\%output_file_B%"
REM Searched 1 file in 2.69 seconds with 7 threads: 1 matching (100%)
REM Searched 32237207 lines: 32236698 matching (100%)




REM METHOD 2 (just some test)
REM New method for Windows (test run - partial (A)): 4.42s total, it doesn't seem to work fast if Tee writes to a file
REM it needs to write to a named pipe, but I failed at "New method for Windows"
%path_ug%\ugrep --encoding=UTF-8 -ayhn --stats -P %phrase% "%path%\%input_file%" | %path_tee%\tee-x64.exe %path%\%temp_file% | %path_ug%\ugrep --stats -P %matched% --format="%1" > "%path%\%output_file_A%"
REM DEL /Q %path%\%temp_file%




REM METHOD 3
REM New method for Windows (attempt to convert METHOD 4 from Linux examples to Windows, and to use named pipes): 
REM maybe can also use %path_tee%\tee-x64.exe, for example:
REM "ugrep | %path_tee%\tee-x64.exe %path_pipes%\createAndWritePipe MyPipeName" but it throws some i/o errors

REM start pipe server (need to remove REM):
REM %path_ug%\ugrep --encoding=UTF-8 -ayhn --stats -P %phrase% "%path%\%input_file%" | %path_pipes%\createAndWritePipe MyPipeName

REM pipe clients won't run because server is blocking processing the current file, and expects clients to be run as separate consoles
REM (via start "" /b), which complicates things, doesn't work as expected, is too slow anyway:
REM %path_ug%\ugrep --stats -P %matched% --format="%1" < \\.\pipe\MyPipeName > "%path%\%output_file_A%"
REM %path_ug%\ugrep --stats -P %unmatched% --format="%1" < \\.\pipe\MyPipeName > "%path%\%output_file_B%"

echo Start Time: %startTime%
echo Finish Time: %time%
PAUSE




REM METHOD 4 (examples from Linux)
REM New method for Linux (examples):

REM ugrep -yhn -options phrase input_file | tee temp_file | ugrep -P '^[0-9]+:(.*)' --format='%1%~' > output_file_A
REM ugrep -P '^[0-9]+-(.*)' --format='%1%~' temp_file > output_file_B

REM mkfifo teepipe
REM ugrep -y -n -F -f words4.txt enwik8 | tee teepipe | ugrep -P '^[0-9]+:(.*)' --format='%1%~' > output1.txt & \
REM ugrep -P '^[0-9]+-(.*)' --format='%1%~' < teepipe > output2.txt
```


</details>

</details>



<details>
<summary>and now my reply [here] to his last comment (SPOILER):</summary>

<br>

>the second troll : https://github.com/Genivia/ugrep/issues/416#issuecomment-2282836170

Since you remove my comments, I reply in my repository:

>the second troll : Your replies are in violation of the code of conduct, which states, among other things:
>the second troll : Your replies do not align with all three.

You've quoted ponts from your "code of conduct" but you still didn't quote sentences / parts of my comments, which are supposed to violate these points, hence your accusations are unfounded. Not to mention you violate these points by yourself, and you accuse me of doing the same at the same time, this is hypocrisy and deliberate trolling.

I already have defeated you on arguments, you didn't provide any meaningful reply or arguments, except blaming me and removing my comments over and over, you are a perfect example of a troll.

> the second troll :  then please find help elsewhere.

I didn't ask you for help, you both need to seek help from a doctor, if you can't control your anger and emotions.

> the second troll : All the best. / Please

Playing / pretending to be nice won't help you, trolls like you and the other guy are good at creating the appearance of being nice and masking their trolling with falsivly nice words, but actually your actions like: blaming, not answering questions, not providing arguments, locking threads, removing comments are the opposite of your false nice-words, your trollish actions speak more than false words. Bye.

</details>

<hr>

One more troll (the third troll):
- https://github.com/stefankueng/grepWin/issues/290#issuecomment-2292569871

<details>
<summary>my reply [here] (SPOILER)</summary>

<br>

I don't know the guy, neither does he know me, I did nothing to him personally, nor I did anything wrong on his repository, he just have seen the drama between me and the two other guys, and he has taken their side, which means - against me:
- he has sticked his personal thumb down emoji on my comment, which (the thumb down emoji) is unrelated to the technical content/matter of my comment, hence his thumb down emoji is his personal hostile off-topic against me
- then he has created an off-topic personal comment against me, which contains some other guy's sentence unsupported by arguments, followed by a link to the external discussion unrelated to the thread, unrelated to him and unrelated to his repository

It was his choice to become my enemy. Since the reason why he did it is unknown, because he didn't provide any arguments or explanation, except stupidly copy&pasting some other guy's sentence unsupported by arguments, there is nothing more to do than to consider him another troll and repay him with thumb down emoji sticked to his stupid comment. I've replied to him here, I'm not even going to reply to him in his repository, because he is a collaborator on his repository, which means he is able to do all the nasty things like the other two guys did to me: trolling, manipulating, hiding/removing my comments, locking a thread, he would just troll me, so I won't give him a chance. 

> You know what I meant. Don't be a nitpicker.

No, I don't know, because you just stupidly pasted a link and quoted some other guys's pointless sentence from a discussion where I defeated him on arguments, and you haven't introduced any new arguments, hence I'm not sure what you wanted to prove, but you proved nothing so far, just like the other guy, except that you proved you have taken his side for unknown reason.

</details>

<hr>

02.10.2024:

Another two trolls, so many them on the whole github:
- https://github.com/uBlockOrigin/uBlock-issues/issues/3359
- https://github.com/uBlockOrigin/uBlock-issues/issues/3362


They were attacking me in their comments and one of them additionally begun to stick thumb down emojis to my comments, I had no other choice than to put them in my blacklist, then they both blocked me too, so I can't reply and defend in their threads anymore, this is what they do when run out of arguments:
![ss](https://github.com/user-attachments/assets/170d1007-20bb-4a4b-86d2-0300aeb00984)

Of course, they didn't inform other people about that I can't reply anymore in their threads, one of them exploited that fact, and posted another comment towards me, hence I've been forced to reply here to his last comment:

> https://github.com/uBlockOrigin/uBlock-issues/issues/3362#issuecomment-2324385100 

<details>
<summary>SPOILER (REPLY)</summary>

> https://github.com/uBlockOrigin/uBlock-issues/issues/3362#issuecomment-2324385100 

> micsthepick : the only thing I found aggressive was "unless you have seen that thread already, and you were inspired by it, and you still want to embed the solution into uBO itself to be even better." not anything else. Is that presumption?

You've just quoted my statement and still didn't explain what is bad/wrong in it, hence I am unable to determine what the hell is your problem.

> micsthepick : It's also a bit ironic that you link to a post where you call someone a clown and yet continue to call me aggressive.

What is ironic is that I called him like that because he was aggressive towards me for no reason, just like you, and he throwed towards me "is everything okay at home, buddy?" which wasn't nice and is considered calling names as well, hence he deserved to be called accordingly as well. I wonder how did you miss that, and how did you cherry-pick only my statement, maybe because you're blind, but I think that rather you're just purposely trolling, you run out of arguments pretty fast, go troll elsewhere, and don't turn facts upside down.

</details>

<hr>

05.10.2024:

Another two trolls, so many them on the whole github:
- https://github.com/uBlockOrigin/uBlock-issues/issues/3359
- https://github.com/uBlockOrigin/uBlock-issues/issues/3362


They were attacking me in their comments and one of them additionally begun to stick thumb down emojis to my comments, I had no other choice than to put them in my blacklist, then they both blocked me too, so I can't reply and defend in their threads anymore, this is what they do when run out of arguments:
![ss](https://github.com/user-attachments/assets/170d1007-20bb-4a4b-86d2-0300aeb00984)

Of course, they didn't inform other people about that I can't reply anymore in their threads, one of them exploited that fact, and posted another comment towards me, hence I've been forced to reply here to his last comment:

> https://github.com/uBlockOrigin/uBlock-issues/issues/3362#issuecomment-2324385100 

<details>
<summary>SPOILER (REPLY)</summary>

> https://github.com/uBlockOrigin/uBlock-issues/issues/3362#issuecomment-2324385100 

> micsthepick : the only thing I found aggressive was "unless you have seen that thread already, and you were inspired by it, and you still want to embed the solution into uBO itself to be even better." not anything else. Is that presumption?

You've just quoted my statement and still didn't explain what is bad/wrong in it, hence I am unable to determine what the hell is your problem.

> micsthepick : It's also a bit ironic that you link to a post where you call someone a clown and yet continue to call me aggressive.

What is ironic is that I called him like that because he was aggressive towards me for no reason, just like you, and he throwed towards me "is everything okay at home, buddy?" which wasn't nice and is considered calling names as well, hence he deserved to be called accordingly as well. I wonder how did you miss that, and how did you cherry-pick only my statement, maybe because you're blind, but I think that rather you're just purposely trolling, you run out of arguments pretty fast, go troll elsewhere, and don't turn facts upside down.

</details>

<hr>

05.10.2024:

Another one (from 3 weeks ago):
- https://github.com/uBlockOrigin/uBlock-issues/discussions/3348

He harassed me, then after I defeated him on argumentation, he withdrew and all the 3 comments were removed, by him or by some other moderator,  hence, there is a backup of the comments: [SCREENSHOOT](https://github.com/user-attachments/assets/102b0960-a090-4be5-8cc5-a0995a23e22d)

<hr>

Another 4 (2 in each thread):
- https://github.com/uBlockOrigin/uBOL-home/issues/196
- https://github.com/uBlockOrigin/uBOL-home/issues/199 + my reply: https://github.com/uBlockOrigin/uBOL-home/issues/201

Where do so many of them come from...

<hr>

This is also interesting:
- https://github.com/uBlockOrigin/uAssets/issues/25063

<hr>

One more to the collection (uBlock creator):

- https://github.com/uBlockOrigin/uBlock-issues/discussions/3023

here is a backup of my comment which I created at the very bottom of the discussion today, and which then has been aggressiviely removed 2 times for nor reason, without any explanation, it's self explanatory (here is also a [SCREENSHOOT](https://github.com/user-attachments/assets/410bffc4-22e6-4f43-b38d-698ea769371b) too):

<details>
<summary>SPOILER </summary>

<br>

> thealio : The problem sussist if you answer with such things like "do it by yourself" or "scratch your own itch", because I'm not a developer, you are, and this kind of answers sound like very silly and not helpful at all.

garry-ut99: True. **gorhill** shouldn't immediately reply with attitude . Such discussions/proposals happen, there is no need go straight to accusation that someone didn't do something himself, this just poisons immediately the discussion. Just pointing out this is not possible or unfeasible is enough.

<hr>

I've noticed just right now that my comment has been removed for no reason without any explanation, hence I've repasted it above again. I don't know why it has been removed, given a fact I only lectured him the same way he lectured me recently: https://github.com/uBlockOrigin/uBOL-home/issues/199#issuecomment-2333973922 (by the way here is my reply to the other guy: https://github.com/uBlockOrigin/uBOL-home/issues/201), why it's suddenly wrong just because now he is the one who is being lectured the same way by his own words, this is hypocrisy and inconsistency, I would expect at most the both comments be moderated equally: both hidden or removed, it seems someone here can't stand truth, and desperately runs out of arguments. Refrain from lecturing others, if you then can't stand being lectured the same way, at least truth has been proved, he can keep removing comments, but he will never remove truth. Nothing for me here it seems, as I'm being purposely trolled here.

</details>

<hr>

Another case (2 users:  a random user + uBlock creator): 
- https://github.com/uBlockOrigin/uBlock-issues/issues/3447

<hr>

Another users trolling: 
`https://github.com/amitbl/blocktube/issues/551`
`https://github.com/amitbl/blocktube/issues/540`
`https://github.com/amitbl/blocktube/issues/535`
`https://github.com/amitbl/blocktube/issues/373`
`https://github.com/notepad-plus-plus/notepad-plus-plus/issues/16658`

<hr>

When a troll is attacking, there is not always a possibility to defend completely, because sometimes threads are locked too fast: before I can put my last reply, also sometimes some comments are unfairly removed or hidden (or unfairly not hidden), also sometimes a troll puts me silently on his blacklist preventing me from defending and commenting in his thread, hence the only possibility to defend for me in such situations is to post links to threads here, and put here my last reply, optionally also a description of what happened, so that people can later read my replies here, and see that I was harassed but I defeated the troll on arguments, otherwise some poeple might believe the troll's nonsense.

**Comment by garry-ut99** [comment]
Two more "aggressive" clowns, so many them on the whole github:
- https://github.com/uBlockOrigin/uBlock-issues/issues/3359
- https://github.com/uBlockOrigin/uBlock-issues/issues/3362

They both banned me so I can't reply in their threads, 
I've been forced to reply here to the second clown comment:

> https://github.com/uBlockOrigin/uBlock-issues/issues/3362#issuecomment-2324385100 : micsthepick : It's also a bit ironic that you link to a post where you call someone a clown and yet continue to call me aggressive.

What is ironic is that I called him a clown because he was aggressive to me for no reason, just like you.
So funny, go troll elsewhere, clown.

**Comment by garry-ut99** [comment]
Two more (in a single thread):
- https://github.com/uBlockOrigin/uBOL-home/issues/196

Where do so many of them come from...

**Comment by garry-ut99** [comment]
Another case:
- https://github.com/uBlockOrigin/uBOL-home/issues/199

**Comment by garry-ut99** [comment]
This thread serves as evidence of attacks on my person by aggressive trolls, from the whole github. 

When a troll is attacking, there is not always a possibility to defend, because sometimes threads are locked too fast: before I can reply, also sometimes some comments are unfairly removed or hidden, also sometimes a troll puts me silently on his blacklist preventing me from defending and commenting in his thread, hence the only possibility to defend for me in such situations is to post links to threads here, and put here my last reply, optionally also a description of what happened, so that people can later read my replies in this thread, and see that I defeat the trolls on arguments, otherwise some poeple might believe the trolls's nonsense.

**Comment by garry-ut99** [comment]
One more to the collection - I've appended it to my previous comment (at the very bottom): https://github.com/garry-ut99/test/issues/1#issuecomment-2332315186

**Comment by garry-ut99** [comment]
One more to the collection:
 - https://github.com/uBlockOrigin/uBlock-issues/discussions/3023

here is a backup of my comment which I created at the very bottom of the linked discussion and which then has been desperately and aggressiviely removed 2 times for nor reason, without any explanation, it's self explanatory. I put my reply here as expanded, to highlight hypocrisy, falsity and inconsistency of that guy, (here is also a [SCREENSHOOT](https://github.com/user-attachments/assets/410bffc4-22e6-4f43-b38d-698ea769371b) too):

<details open>
<summary>SPOILER (REPLY BACKUP)</summary>

<br>

>> thealio : The problem sussist if you answer with such things like "do it by yourself" or "scratch your own itch", >because I'm not a developer, you are, and this kind of answers sound like very silly and not helpful at all.
>
>garry-ut99: True. **gorhill** shouldn't immediately reply with attitude . Such discussions/proposals happen, there is no need go straight to accusation that someone didn't do something himself, this just poisons immediately the discussion. Just pointing out this is not possible or unfeasible is enough.
>
> <hr>
>
> I've noticed just right now that my comment has been removed for no reason without any explanation, hence I've repasted it above again. I don't know why it has been removed, given a fact I only lectured him the same way he lectured me recently: https://github.com/uBlockOrigin/uBOL-home/issues/199#issuecomment-2333973922, why it's suddenly wrong just because now he is the one who is being lectured the same way by his own words, this is hypocrisy and inconsistency, I would expect at most the both comments be moderated equally: both hidden or removed, it seems someone here can't stand truth, and desperately runs out of arguments. Refrain from lecturing others, if you then can't stand being lectured the same way, at least truth has been proved, he can keep removing comments, but he will never remove truth. Nothing for me here it seems, as I'm being purposely trolled here.

</details>

**Comment by garry-ut99** [comment]
22.11.2024:

He (uBlock creator) now suddenly has banned me from the whole uBlockOrigin project without providing any reason or argument:
- I can't write any comment anymore:
![ss1](https://github.com/user-attachments/assets/6d511403-47c9-40d5-ba4d-764ae9685621)
- but this was insufficient for him, in his uncontrolled anger and emotional/mental unstability, he additionally has vandalised and hide all my comments for no reason, several hundreds useful technical comments on 3 different repositories, which were useful for hundreds / thousands of uBO users:
   - [uBlockOrigin/uBlock-issues]( https://github.com/uBlockOrigin/uBlock-issues/issues?q=sort%3Aupdated-desc+involves%3Agarry-ut99) + [uBlock-issues/discussions](https://github.com/uBlockOrigin/uBlock-issues/discussions?discussions_q=involves%3Agarry-ut99)
   - [uAssets/issues](https://github.com/uBlockOrigin/uAssets/issues?q=sort%3Aupdated-desc+involves%3Agarry-ut99) + [uAssets/discussions](https://github.com/uBlockOrigin/uAssets/discussions?discussions_q=involves%3Agarry-ut99)
   - [uBOL-home/issues](https://github.com/uBlockOrigin/uBOL-home/issues?q=sort%3Aupdated-desc+is%3Aissue+involves%3Agarry-ut99) + [uBOL-home/discussions](https://github.com/uBlockOrigin/uBOL-home/discussions?discussions_q=involves%3Agarry-ut99)
![ss3](https://github.com/user-attachments/assets/030852d4-cfe1-4f70-9787-238707ef3512)
![ss2](https://github.com/user-attachments/assets/45f747b3-2cce-4b8a-84e3-4c46484d3660)
- but this was still insufficient for him, he has gone even further in his vandalism - as the reason for hiding them he selected inappropriate, deceitful labels: "abuse" or "disruptive content" despite that hundreds of my comments are not "abuse" nor disruptive", and they stayed visible for months and years, and some of them received many thumbs up emojis from other users to which they were useful, which proves that they (comments) never were neither "abuse" nor "disruptive", he has just falsively turned several hundreds of my comments into "abuse" and "disruptive" in a matter of a few seconds. In such a case, it's his actions which are "abuse" / "destructive", and not my comments.
- of course he did all his malicious actions silently, without informing the whole uBlock community about anything

Banning + hiding several comments is sufficient, if he really needs to, but hiding and untruthfully labeling hundreds of useful technical comments is pure vandalism and mass spamming/flooding. The github admin feature to bulk hide and label hundreds of comments is to prevent spam-bots or spam-users, not to abuse it for private vendettas.

He lacks an argument, he shouldn't immediately act with attitude and ban long time well-deserved contributors like this, there is no need go straight to ban users and to vandalise hundreds of their comments for no reason by hiding them and labeling them falsively by inappropriate, deceitful labels. Such furious actions just poison immediately the whole uBO project and harm users who seek advice and help. He should stop escalating and should de-escalate instead, there was no need for such escalation from his side. His actions are disruptive, and this was not the first time he acted like this. By the way, I don't need to be another monkey in his circus.

<hr>

30.03.2025:

He (uBlock creator aka "gorhill") again has attacked and vandalized several comments for no reason (like already described in previous comment https://github.com/garry-ut99/test/issues/1#issuecomment-2468932596) and interrupted a peaceful technical conversation.

In https://github.com/uBlockOrigin/uBlock-issues/issues/3594 he has hidden and vandalized 3 comments and fake-labeled them as "off-topic", and locked the conversation, despite none of them are off-topic, it's actually the opposite: they are perfectly on-topic, technical and useful comments. He definitely abuses his moderator privileges, by his toxic actions. 

"Be better, instead of being a j.....ss to people that are giving you their free time to help your project."

**Comment by garry-ut99** [comment]
Another one: `https://github.com/amitbl/blocktube/issues/535`.

**Comment by garry-ut99** [comment]
He (uBlock creator aka "gorhill") is continuing to troll and vandalize comments for no reason (like already described in previous comment https://github.com/garry-ut99/test/issues/1#issuecomment-2468932596).

He now has aggressively interrupted peaceful conversation and has devastated / vandalised / trolled another comments in another thread for no reason: https://github.com/uBlockOrigin/uBlock-issues/issues/3594 - he hide 3 comments as off-topic and locked the conversation, despite none of them are off-topic, it's actually the opposite: they are **perfectly on-topic**, technical and useful comments. He definitely abuses his moderator privileges, he is very aggressive, toxic, disrespectful and he just continues to vandalise comments for no reason. 

Be better, instead of being a jackass to people that are giving you their free time to help your project.

@gwarser, what is this rude crazy monkey doing? Why did he disrespectfully and agressivaly interruped or peaceful technical conversation and vandalised our comments?
